grpc.go

```diff
@@ -550,6 +550,7 @@ func (a *agentGRPC) SignalProcess(ctx context.Context, req *pb.SignalProcessRequ
 			"sandbox":     a.sandbox.id,
 			"signal":      signal.String(),
 		}).Info("discarding signal as container stopped")
+		return nil, nil
```

Comment:
I think the first value returned here should be `emptyResp` for consistency.

Reply:
Correct.

Reply:
Yes @vbmade2000 please do this change.

Reply:
Done